src/modules/playRoom/router/index.js

```diff
@@ -7,7 +7,7 @@ const authRoutes = [
     name: 'PlayRoom',
     component: PlayRoom,
     meta: {
-      requireAuth: true,
+      requireAuth: false,
```

Comment:
only do this while you are developing or testing. Do not commit it to your pull request unless you sure know what you are doing